torch/testing/_internal/common_nn.py

```diff
@@ -4523,6 +4536,16 @@ def padding3d_circular(input, pad):
         desc='dim_is_3',
         check_bfloat16=False,
     ),
+    dict(
+        module_name='CrossEntropyLossWithSoftLabels',
+        input_size=(5, 3),
+        target_fn=lambda: torch.rand(5, 3),
+        reference_fn=lambda i, t, m:
+            loss_reference_fns['CrossEntropyLossWithSoftLabels'](i, t, reduction=get_reduction(m)),
+        check_sum_reduction=True,
+        desc='2d',
+        check_bfloat16=False,
+    ),
```

Comment:
We should test all of the reduction options. Does this one only test `reduction='sum'`, or does it test all three? (sum, mean, none)

Reply:
Unfortunately it looks like it only tests `reduction='sum'` :( (why this does exist? haha) I'll add some manual tests in `test_nn.py` for the other cases.